### PR TITLE
Links to Student previews in the Instructor view for the Assignment

### PR DESCRIPTION
**Pull request by Evan-CTL**

### CODE REVIEW

File: econplayground/templates/assignment/assignment_detail.html [hunk 1]
```diff
@@ -20,6 +20,12 @@
 
 {% block pagetitle %}
     <div class="float-end">
+        <a class="btn btn-secondary display-inline mx-2"
+           role="button"
+           href="{% url 'assignment_student' object.pk %}"
```
Comment (nikolas) [inline econplayground/templates/assignment/assignment_detail.html:25]: The views that these links link to are not used in the current assignment setup - I suppose I need to clean this up.

Comment (Evan-CTL) [reply econplayground/templates/assignment/assignment_detail.html:25]: Where would you like me to point them to?

Comment (nikolas) [reply econplayground/templates/assignment/assignment_detail.html:25]: For the assignment detail preview, we have the links to each Step, which is essentially student preview. We don't really have "main page" for the assignment on the student side yet, but I'll be putting that together soon.

For previewing what the student sees on the assignment list, that will depend on which courses the student is part of, which is still to be determined, but i'll be working on that surrounding stuff this sprint.

Comment (Evan-CTL) [reply econplayground/templates/assignment/assignment_detail.html:25]: Okay, I'll hold off on updating this then

**Pull request merged**